### PR TITLE
Edits getting started

### LINKED ISSUE #455: Edit Getting Started Guide
- Procedures are ULs instead of OLs
 - more links

Part of #402

### PR DESCRIPTION
**Pull request by adammichaelwood**
closes #455 

## What is included in this PR?

Moved the section about how ODK gets used to the home page,
and cleaned up Getting Started for style, better internal links, and clearer procedure.

### CODE REVIEW

File: odk1-src/getting-started.rst [hunk 1]
```diff
@@ -1,90 +1,81 @@
...
 
-For other installation options, see :doc:`collect-install`.
+.. seealso:: :doc:`collect-install`
 
-.. _installing-aggregate:
+.. _getting-started-install-aggregate:
 
-Install Aggregate
----------------------
+Install Aggregate (optional)
```
Comment (yanokwa) [inline odk1-src/getting-started.rst]: If you don't install Aggregate, you can't do the things in this list (e.g., create a form with Build and upload it to Aggregate and so on. 

I think maybe we should have different levels? So Level 1 would be to install Collect, fill out a form, and see it on Aggregate. Then Level 2 would be to a form, upload it to a server, download it to Collect, fill it out, and see it on Aggregate.

The flaw in this plan is that it's not so easy to install Aggregate. And even with the VM it's not that fun to get Collect/Aggregate integration. So maybe we need to have a sandbox server that we delete data from nightly?

**Comment by yanokwa** [comment]
@adammichaelwood I've set up a sandbox at https://sandbox.aggregate.opendatakit.org:8443. It needs some spit and polish, but should at least let people try stuff.

**Comment by adammichaelwood** [comment]
When I try the sandbox link, I get a time out error.

**Comment by adammichaelwood** [comment]
I've updated this to link to the sandbox server. 
Do you think it should be broken out even more (as suggested in your previous comment) or does this work?

File: odk1-src/getting-started.rst [hunk 2]
```diff
@@ -1,90 +1,88 @@
...
-Install Collect
----------------------
+The easiest way to set up Aggregate is to
+:doc:`install it on Google App engine <aggregate-app-engine>`. 
 
-The easiest way to install the ODK Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
+You'll set up a new Google Cloud project, and then run the install utility locally. This will connect to your Google Cloud account and install Aggregate there.
 
-For other installation options, see :doc:`collect-install`.
+Alternatively, if you just want to try things out,
```
Comment (yanokwa) [inline odk1-src/getting-started.rst]: s/just/g

File: odk1-src/getting-started.rst [hunk 3]
```diff
@@ -1,90 +1,88 @@
...
+Install Aggregate (optional)
+------------------------------
 
-Install Collect
----------------------
+The easiest way to set up Aggregate is to
+:doc:`install it on Google App engine <aggregate-app-engine>`. 
 
-The easiest way to install the ODK Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
+You'll set up a new Google Cloud project, and then run the install utility locally. This will connect to your Google Cloud account and install Aggregate there.
```
Comment (yanokwa) [inline odk1-src/getting-started.rst]: s/install utility/installer/

File: odk1-src/getting-started.rst [hunk 4]
```diff
@@ -1,90 +1,88 @@
...
 
-For other installation options, see :doc:`collect-install`.
+Alternatively, if you just want to try things out,
+you can use the `Aggregate sandbox server`_.
 
-.. _installing-aggregate:
+.. _Aggregate sandbox server: https://sandbox.aggregate.opendatakit.org:8443
```
Comment (yanokwa) [inline odk1-src/getting-started.rst]: We should add a warning that `This is a demonstration server. All forms and data on this server are public and are occasionally deleted.`

File: odk1-src/getting-started.rst (hunk 2 above)
Comment (adammichaelwood) [reply odk1-src/getting-started.rst]: What about "only".

Comment (yanokwa) [reply odk1-src/getting-started.rst]: Only works. Just is bad because it makes it sound easy and it might not be easy for someone.

Comment (adammichaelwood) [reply odk1-src/getting-started.rst]: Right.

**Comment by adammichaelwood** [comment]
> Shameful delay in reviewing, but it's done.

Let he who has done any actual work in the last two weeks,
cast the first stone.

**Pull request merged**